hikari/impl/rest.py

```diff
@@ -416,7 +415,7 @@ def __init__(
         max_rate_limit: float,
         proxy_settings: config.ProxySettings,
         token: typing.Union[str, None, rest_api.TokenStrategy],
-        token_type: typing.Union[applications.TokenType, str, None],
+        token_type: typing.Union[applications.TokenType, str],
```

Comment:
What should be passed when the token is a Token Strategy or Nonr

Reply:
Good point, will fix